### PR TITLE
Fixing images

### PR DESCRIPTION
**Pull request by travelar**
Fixed URL for images.

### CODE REVIEW

File: security/yubikey-configuration.md [hunk 1]
```diff
@@ -136,11 +136,11 @@ This subkey will be used to pull private git repos via SSH and may be used to au
 This is interface has a unique design where you need to toggle things on and off to get the desired result.
 The default state shows **Sign Encrypt** active.
 
-![Sign Encrypt Default State](https://github.com/trussworks/Engineering-Playbook/blob/yubikey-guide/security/images/Yubikey-Sign-Encrypt.png "default state")
+![Sign Encrypt Default State](https://github.com/trussworks/Engineering-Playbook/tree/master/security/images/Yubikey-Sign-Encrypt.png "default state")
```
Comment (rpdelaney) [inline security/yubikey-configuration.md]: Relative paths should work I _think_ ...

Comment (rpdelaney) [inline security/yubikey-configuration.md]:
```suggestion
![Sign Encrypt Default State](images/Yubikey-Sign-Encrypt.png "default state")
```

Comment (cblkwell) [reply security/yubikey-configuration.md]: yeah, I think ryan might have the right idea here...

Comment (travelar) [reply security/yubikey-configuration.md]: Ahhh.. yeah. I think that's better.

**Pull request merged**